deployment/overlays/samples/cert-manager/probes.yaml

```
@@ -0,0 +1,8 @@
+- op: replace
+  path: /spec/template/spec/containers/0/livenessProbe/exec/command
+  value: ["/usr/bin/grpc_health_probe", "-addr=:8080", "-tls", "-tls-ca-cert=/etc/kubernetes/node-feature-discovery/certs/ca.crt", "-tls-client-key=/etc/kubernetes/node-feature-discovery/certs/tls.key", "-tls-client-cert=/etc/kubernetes/node-feature-discovery/certs/tls.crt"]
```

nit: could we use `add` (instead of `replace`), similarly what we do in `args.yaml`? I think this "additive" method is probably less prone to problems e.g. if something in the base yaml would change.

---

Agree.  I wanted the more simpler patch to begin with, but I agree that to add the new args on the end would be more future proof.

---

I think it would be easier to just change it in this PR. I think we could merge this after that

```suggestion
- op: add
  path: /spec/template/spec/containers/0/livenessProbe/exec/command/-
  value: "-tls"
- op: add
  path: /spec/template/spec/containers/0/livenessProbe/exec/command/-
  value: "-tls-ca-cert=/etc/kubernetes/node-feature-discovery/certs/ca.crt"
- op: add
  path: /spec/template/spec/containers/0/livenessProbe/exec/command/-
  value: "-tls-client-key=/etc/kubernetes/node-feature-discovery/certs/tls.key"
- op: add
  path: /spec/template/spec/containers/0/livenessProbe/exec/command/-
  value: "-tls-client-cert=/etc/kubernetes/node-feature-discovery/certs/tls.crt"
```